server/src/main/java/keywhiz/service/permissions/AlwaysFailPermissionCheck.java

```diff
@@ -9,4 +9,6 @@ public AlwaysFailPermissionCheck() {}
   public boolean isAllowed(Object source, String action, Object target) {
```

Comment:
Will the AlwaysFailPermissionCheck be used anywhere?

Reply:
I think it's currently only used in tests, but I believe we originally implemented it for completeness (if we have always allow, we should have always deny, essentially).